feature-libs/cart/main/core/http-interceptors/handlers/bad-cart-request.handler.ts

```diff
@@ -44,6 +45,22 @@ export class BadCartRequestHandler extends HttpErrorHandler {
       });
   }
 
+  protected handleOtherCartErrors(
+    _request: HttpRequest<any>,
+    response: HttpErrorResponse
+  ): void {
+    this.getErrors(response)
+      .filter((e) => e.reason !== 'notFound' || e.subjectType !== 'cart')
```

Comment:
instead of `e.reason !== 'notFound' || e.subjectType !== 'cart')`, could we use ` !isCartNotFoundError(e)`

Reply:
`isCartNotFoundError(e)` also check `!isSelectiveCart(error.subject)`